specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/AccessControlLists.json

```diff
@@ -645,6 +645,12 @@
           "format": "uri",
           "minLength": 1
         },
+        "defaultAction": {
+          "type": "string",
```

Comment:
>string [](http://example.com/codeflow?start=19&length=6)

If the set of values are pre-defined, make it an enum?

Reply:
I have referred Enum, **CommunityActionTypes**